src/GitLink/Helpers/ProjectHelper.cs

```diff
@@ -29,7 +29,7 @@ public static class ProjectHelper
 
         static ProjectHelper()
         {
-            SolutionParserType = TypeCache.GetType("Microsoft.Build.Construction.SolutionParser");
+            SolutionParserType = TypeCache.GetType("Microsoft.Build.Construction.SolutionParser, Microsoft.Build, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a");
```

Comment:
well, ya. that's why it works. Without qualifying the type with the assembly that provides it (you compile against MSBuild 4.0), the CLR has no idea where to find it unless the assembly is already loaded. With my change, the CLR will load the assembly on-demand, ensuring it works every time.